test/Libraries/Microsoft.Extensions.Telemetry.Tests/Logging/ExtendedLoggerTests.cs

```diff
@@ -431,18 +453,18 @@ public static void Exceptions(bool includeExceptionMessage)
 
         var state = snap[3].StructuredState;
```

Comment:
nit: This variable is used only once on line 477 and can be inlined there.

Reply:
Done.